src/atoms/Typography.tsx

```diff
@@ -0,0 +1,28 @@
+import type { FunctionComponent } from 'react';
+
+import type { TextProps } from '.';
+import { Text } from '.';
+
+export const Heading: FunctionComponent<TextProps> = ({ children, as = 'h1', ...props }) => (
+  <Text as={as} variant="heading" {...props}>
+    {children}
+  </Text>
+);
+
+export const SubHeading: FunctionComponent<TextProps> = ({ children, as = 'h2', ...props }) => (
+  <Text as={as} variant="subHeading" {...props}>
+    {children}
+  </Text>
+);
+
+export const Body: FunctionComponent<TextProps> = ({ children, as = 'p', ...props }) => (
```

Comment:
In `core` defaulting to a paragraph is actually troublesome because of the default bottom margin.
@guillaume @Mrtenz should we:
- remove the default `p` margin (if it's important for semantics)
- set the default to a `inline-block` `span`

Reply:
I think core is the exception in this case, since the signer, landing page, and knowledge base all use paragraphs with bottom margin. We could do a separate inline text component?

Reply:
Fine by me.

Reply:
I'm okay with it